Gemfile

```diff
@@ -2,6 +2,7 @@ source 'https://rubygems.org'
 
 gem 'cocoapods'
 gem 'cocoapods-update-if-you-dare', '~> 0.1.1'
+gem 'colored'
```

Comment:
Ah, thanks, @garriguv, I've missed the commit description 😄 And thanks for the PR to the CocoaPods plugin 👍 Maybe the @Ashton-W can merge it soon and release a new version, so we again won't require the `colored` dependency specified in the gem.